clients/src/test/java/org/apache/kafka/clients/admin/KafkaAdminClientTest.java

```diff
@@ -4846,6 +4878,31 @@ public void testDescribeFeaturesFailure() {
         }
     }
 
+    @Test
+    public void testDescribeMetadataQuorumSuccess() throws Exception {
+        try (final AdminClientUnitTestEnv env = mockClientEnv()) {
+            env.kafkaClient().setNodeApiVersions(NodeApiVersions.create((short) 55, (short) 0, (short) 1));
```

Comment:
ApiKeys.DESCRIBE_QUORUM.id,  DESCRIBE_QUORUM.highestSupportVersion， DESCRIBE_QUORUM.lowestSupportVersion may be better here.